Framework/Geometry/src/Objects/ShapeFactory.cpp

```diff
@@ -163,11 +163,6 @@ ShapeFactory::createShape(Poco::XML::Element *pElem) {
             lastElement = pE;
             idMatching[idFromUser] = parseCylinder(pE, primitives, l_id);
             numPrimitives++;
-          } else if (primitiveName == "segmented-cylinder") {
```

Comment:
Can we not support this anymore?

Reply:
We never truly supported it. It was the same as the normal cylinder.

Reply:
Fair enough - just wanted to double check :+1: